src/Glpi/Form/Migration/FormMigration.php

```diff
@@ -332,6 +356,8 @@ protected function processMigration(): bool
         $this->processMigrationOfTranslations();
         $this->processMigrationOfConditions();
 
+        $this->displayWarningForNonMigratedItems();
+
         $this->progress_indicator?->setProgressBarMessage('');
         $this->progress_indicator?->finish();
```

Comment:
I think we can consider the proress is finished before the warning display that is part of the result.

```suggestion
        $this->progress_indicator?->setProgressBarMessage('');
        $this->progress_indicator?->finish();

        $this->displayWarningForNonMigratedItems();
```